test/E2ETest/WebStack.QA.Test.OData/SxS/ODataV3/Controllers/ProductsController.cs

```diff
@@ -1,40 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Web.Http;
-using System.Web.Http.OData;
-using WebStack.QA.Test.OData.SxS.ODataV3.Models;
-
```

Comment:
why do you delete these test cases?

Is there any plan to re-add them

Reply:
I don't know if I fully understand the value of these tests but I also don't have a food reason to delete them. However, they will not be ported to NetCore since we don't have a V3 NetCore compatible version.